yb-voyager/cmd/templates/schema_analysis_report.txt

```diff
@@ -2,9 +2,10 @@
 |               Database Migration Report         |
 +-------------------------------------------------+
 
-Database Information
+Migration Information
```

Comment:
I am not sure if we should be changing `Database Information` -> `Migration Information`

Most of the info in it are just Database info only, except Migration Complexity of the database...

Reply:
Even Voyager version cant be put under database head. Keeping it as migration until we find a better head.